runtime/lua/vim/filetype.lua

```diff
@@ -770,6 +770,14 @@ local extension = {
   mm = function() vim.fn["dist#ft#FTmm"]() end,
   mms = function() vim.fn["dist#ft#FTmms"]() end,
   p = function() vim.fn["dist#ft#FTprogress_pascal"]() end,
+  patch = function(path, bufnr)
+    local firstline = vim.fn.getline(1)
```

Comment:
Use the `getline` helper function from this file:

```suggestion
    local firstline = getline(bufnr, 1)
```